README.rst

```diff
@@ -1,130 +1,165 @@
+bears themselves,
+`click here. <https://github.com/coala-analyzer/bear-docs/blob/master/README.rst>`__
+
+|Linux Build Status| |Windows Build status| |Scrutinizer Code Quality|
+|codecov.io| |Documentation Status| |Gitmate|
 
 .. Start ignoring LineLengthBear
 
-::
+================================================= ================================================ ====================================================== =========================================================
+`Official Website <http://coala-analyzer.org/>`__ `Twitter <https://twitter.com/coala_analyzer>`__ `Facebook <https://www.facebook.com/coalaAnalyzer/>`__ `Video Demo <https://asciinema.org/a/42968?autoplay=1>`__
+================================================= ================================================ ====================================================== =========================================================
 
-                                                         .o88Oo._
-                                                        d8P         .ooOO8bo._
-                                                        88                  '*Y8bo.
-                                          __            YA                      '*Y8b   __
-                                        ,dPYb,           YA                        68o68**8Oo.
-                                        IP'`Yb            "8D                       *"'    "Y8o
-                                        I8  8I             Y8     'YB                       .8D
-                                        I8  8P             '8               d8'             8D
-                                        I8  8'              8       d8888b          d      AY
-         ,gggo,    ,ggggo,    ,gggo,gg  I8 dP    ,gggo,gg   Y,     d888888         d'  _.oP"
-        dP"  "Yb  dP"  "Y8go*8P"  "Y8I  I8dP    dP"  "Y8I    q.    Y8888P'        d8
-       i8'       i8'    ,8P i8'    ,8I  I8P    i8'    ,8I     "q.  `Y88P'       d8"
-      ,d8,_    _,d8,   ,d8' d8,   ,d8b,,d8b,_ ,d8,   ,d8b,       Y           ,o8P
-    ooP""Y8888PP*"Y8888P"   "Y8888P"`Y88P'"Y88P"Y8888P"`Y8            oooo888P"
+.. Stop ignoring
 
-.. Stop ignoring LineLengthBear
+-----
 
-Get *coala* to lint all your languages in your project with one tool and
-config!
+.. contents::
+    :local:
+    :depth: 1
+    :backlinks: none
 
-Demo (Click to View)
---------------------
+-----
 
-|asciicast|
+========
+Features
+========
 
-.. |asciicast| image:: https://asciinema.org/a/42968.png
-   :target: https://asciinema.org/a/42968?autoplay=1
-   :width: 100%
+* Out-of-the-box support for various `popular languages <https://github.com/coala-analyzer/bear-docs/blob/master/README.rst>`__
+  such as **C/C++**, **Python**, **Javascript**, **CSS**, **Java** and many
+  others with built-in check routines.
+* User-friendly interfaces such as JSON, interactive CLI or any custom format.
+* Plugins for gedit, Sublime Text, Atom, Vim and Emacs.
```

Comment:
I think a link to the coala-gedit repo is better since a gedit user has no use to a link to the gedit home page :P